py3-pandas.yaml

```diff
@@ -54,6 +54,7 @@ subpackages:
         - py${{range.key}}-numpy
         - py${{range.key}}-pytz
         - py${{range.key}}-dateutil
+        - py${{range.key}}-tzdata
     pipeline:
```

Comment:
Should it be here, or runtime dep on py3-pytz packages?

Reply:
It should be here. It's specifying the deps for the correct python version in each package runtime deps. The top level package `py3-pytz` is really just an empty package and I wish we didn't have to generate it all the magic is done by the subpackages provides.

Reply:
indeed they do conditional direct import at https://github.com/pandas-dev/pandas/blob/f598670353311a6fff4e6e1e96074ccf0737e6b7/pandas/_libs/tslibs/timezones.pyx#L64